### PR TITLE
Replace "foo:bar" in the secret creation view with a placeholder

### PR DESCRIPTION
**Pull request by susanreenesaa**
#1572
WIP

Bugzilla Bug: [XXXXX](https://bugzilla.mozilla.org/show_bug.cgi?id=XXXXX)

### CODE REVIEW

File: ui/src/components/SecretForm/index.jsx [hunk 1]
```diff
@@ -225,12 +225,22 @@ export default class SecretForm extends Component {
...
                 />
+<!--              <CodeEditor
+                  onChange={this.handleEditorChange}
+                  mode="yaml"
+                  lint
+                  value={editorValue}
+                /> -->
               </ListItem>
             )}
           </List>
```
Comment (helfi92) [inline ui/src/components/SecretForm/index.jsx]: Try removing these changes and replacing them with:

```diff
diff --git a/ui/src/components/SecretForm/index.jsx b/ui/src/components/SecretForm/index.jsx
index 8245b7127..e8a124385 100644
--- a/ui/src/components/SecretForm/index.jsx
+++ b/ui/src/components/SecretForm/index.jsx
@@ -106,9 +106,7 @@ export default class SecretForm extends Component {
       ? addYears(new Date(), 1000)
       : this.props.secret.expires,
     editorValue: this.props.isNewSecret
-      ? safeDump({
-          foo: 'bar',
-        })
+      ? ''
       : safeDump(this.props.secret.secret),
     showSecret: this.props.isNewSecret,
   };
@@ -226,6 +224,7 @@ export default class SecretForm extends Component {
             {showSecret && (
               <ListItem className={classes.editorListItem}>
                 <CodeEditor
+                  placeholder="foo:bar"
                   onChange={this.handleEditorChange}
                   mode="yaml"
                   lint
```

Does that work?

File: ui/src/components/SecretForm/index.jsx [hunk 2]
```diff
@@ -225,12 +225,22 @@ export default class SecretForm extends Component {
...
+                  label="Enter secret"
+                  name="editorValue"
                   onChange={this.handleEditorChange}
                   mode="yaml"
                   lint
-                  value={editorValue}
+                  fullWidth
+                  autoFocus
                 />
+<!--              <CodeEditor
```
Comment (helfi92) [inline ui/src/components/SecretForm/index.jsx]: https://wesbos.com/react-jsx-comments/

**Comment by djmitche** [comment]
If you rebase onto the latest master, it will fix the test errors.

File: ui/src/components/SecretForm/index.jsx (hunk 1 above)
Comment (susanreenesaa) [reply ui/src/components/SecretForm/index.jsx]: okay, thank you.

**Comment by susanreenesaa** [comment]
@djmitche. I can nologer rebase my master branch.  after typing the rebase command, whenever I enter my github username and password, it throws an error of invalid credentials yet they are the correct ones. but that happened after filling the Two Factor..... the one that provides recovery Github codes.
I will be grateful for your help so that I can rebase the master branch

**Comment by djmitche** [comment]
The typical way to push changes to github is to use SSH -- [This guide](https://help.github.com/en/articles/connecting-to-github-with-ssh) describes how to set that up.  That way, you don't need to type a username and password (and MFA code) all the time.

**Comment by susanreenesaa** [comment]
thanks @djmitche and @helfi92

File: ui/src/components/SecretForm/index.jsx [hunk 3]
```diff
@@ -226,6 +224,7 @@ export default class SecretForm extends Component {
             {showSecret && (
               <ListItem className={classes.editorListItem}>
                 <CodeEditor
+                  placeholder="foo:bar"
```
Comment (djmitche) [inline ui/src/components/SecretForm/index.jsx]: Perhaps "YAML representation of secret data"?

Comment (helfi92) [inline ui/src/components/SecretForm/index.jsx]: Agreed with @djmitche, let's do `placeholder="YAML representation of secret data"`,

Comment (susanreenesaa) [reply ui/src/components/SecretForm/index.jsx]: okay. let me do that.

**Comment by helfi92** [comment]
@all-contributors please add @susanreenesaa for code.

**Pull request merged**